pom.xml

```diff
@@ -45,7 +45,6 @@
     <commonslang3.version>3.14.0</commonslang3.version>
     <commonslogging.version>1.3.1</commonslogging.version>
     <commonstext.version>1.11.0</commonstext.version>
-    <fasterxml.version>2.16.1</fasterxml.version>
```

Comment:
Deleted as we are using shaded Jackson from JDBC.

Reply:
The fasterxml.Jackson is only used internally in SDK for serialize & deserialize json. As long as the serializer version is the same as the deserializer, it should be fine.

Reply:
Left a comment on another thread for basically the same thing:
 https://github.com/snowflakedb/snowflake-ingest-java/pull/773#discussion_r1645244609